LayoutTests/media/media-source/media-managedmse-bufferedchange.html

```diff
@@ -37,6 +37,10 @@
             run('sourceBuffer.appendBuffer(loader.initSegment())');
             await waitFor(sourceBuffer, 'update');
 
+            sourceBuffer.onbufferedchange = () => {
```

Comment:
it would have failed if the property was missing.
That's how I found the bug to start.

If the sourceBuffer object is actually a `SourceBuffer` rather than a `ManagedSourceBuffer` then the `onbufferedchange` property doesn't exist to start, and so setting it does nothing.

Reply:
It wouldn't have failed, it would have given *different* output which could have easily been mistakenly rebaselined. It is bad practice. Please make it so that the test output contains a FAIL line if the property is missing.